app/app/tests/utils/utils.py

```diff
@@ -37,14 +37,15 @@ def get_superuser_token_headers(client: TestClient, type_: str = 'access') -> Di
     return headers
 
 
-def to_json(obj: Any) -> Dict:
-    has_relation = len(obj.__mapper__.relationships.items()) > 0
+def to_json(obj: Any, show_relations: Optional[bool] = True) -> Dict:
+    if show_relations and len(obj.__mapper__.relationships.items()) == 0:
```

Comment:
won't it be better to directly check this below instead of resetting `show_relations`

sorry for the double comment, I deleted the older one by mistake

Reply:
hmm tru